scripts/InstallNet4.ps1

```diff
@@ -0,0 +1,52 @@
+$NetFx4ClientUrl = 'http://download.microsoft.com/download/5/6/2/562A10F9-C9F4-4313-A044-9C94E0A8FAC8/dotNetFx40_Client_x86_x64.exe'

```

Comment:
Similar question to above.... where did this requirement come from?

Reply:
Resolved via Zoom.